src/main/java/com/github/admissionCommittee/web/controllers/AdminController.java

```diff
@@ -0,0 +1,32 @@
+package com.github.admissionCommittee.web.controllers;
+
+import com.github.admissionCommittee.service.UserService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet({"/admin"})
+public class AdminController extends HttpServlet {
+    private static final Logger log = LoggerFactory.getLogger(LoginController.class);
+    private UserService service;
+
+    @Override
+    public void init(ServletConfig config) throws ServletException {
+        service = (UserService) config.getServletContext().getAttribute("userService");
+    }
```

Comment:
it's better to call also super.init() when you override something in standard library